changelogs/fragments/fix-build-files-manifest-walk.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - galaxy - Make sure paths are correct building collections when runnning ``_build_files_manifest_walk``
```

There's no triple "n" in "runnning" :)
I think we typically the CLI command name in changelog, also let's add more clarity:
```suggestion
  - >-
    ansible-galaxy - Fixed processing trailing directory separator when building collections
    (https://github.com/ansible/ansible/pull/81619).
```